extensions/wikia/PhalanxII/fallback/hooks/ContentBlock.class.php

```diff
@@ -15,42 +15,42 @@ public static function onEditFilter( $textbox, $summary, &$block ) {
 		global $wgOut, $wgTitle;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$wgOut" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)